app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
```

Comment:
Ensure that the `WEATHER_API_KEY` environment variable is set before running the script. Consider adding error handling to check if `API_KEY` is `None` and provide a user-friendly error message if it is not set.